Tasks/AzureResourceGroupDeployment/operations/Utils.ts

```diff
@@ -10,6 +12,17 @@ class Utils {
         }
         return error;
     }
+
+    public static async getTargetUriFromFwdLink(fwdLink: string) {
```

Comment:
You should consider adding this to a utils file in the common folder. It can be reused by any other task too, right?